deepchem/models/tests/test_ferminet.py

```diff
@@ -35,7 +35,7 @@ def test_forward():
     FH_molecule = [['F', [0.424, 0.424, 0.23]], ['H', [0.4, 0.5, 0.5]]]
     # Testing ionic initialization
     mol = FerminetModel(FH_molecule, spin=1, ion_charge=-1)
-    result = mol.model.forward(mol.molecule.x)
+    result = mol.model.forward(torch.from_numpy(mol.molecule.x))
```

Comment:
model will accept torch tensor in this new change thus making it possible for hessian and jacobian calculation without the previous nested loop